app/common/renderer/components/Inspector/SavedGestures.jsx

```diff
@@ -1,13 +1,21 @@
-import {DeleteOutlined, EditOutlined, PlayCircleOutlined, PlusOutlined} from '@ant-design/icons';
-import {Button, Space, Table, Tooltip} from 'antd';
+import {
+  DeleteOutlined,
+  EditOutlined,
+  PlayCircleOutlined,
+  PlusOutlined,
+  DownloadOutlined,
+} from '@ant-design/icons';
+import {Button, Collapse, Modal, Row, Space, Table, Tooltip} from 'antd';
 import _ from 'lodash';
 import moment from 'moment';
 import React, {useEffect, useRef} from 'react';
 
 import {POINTER_TYPES, SAVED_GESTURE_PROPS} from '../../constants/gestures';
 import {SCREENSHOT_INTERACTION_MODE} from '../../constants/screenshot';
-import {percentageToPixels} from '../../utils/other';
+import {downloadFile, percentageToPixels} from '../../utils/other';
 import InspectorStyles from './Inspector.module.css';
+import FileUploader from './FileUploader.jsx';
+import {ExclamationCircleOutlined} from '@ant-design/icons';
```

Comment:
This import can be merged with the import on line 1

Reply:
My IDE is messing with the imports. Fixed it now!